src/containers/Dev/DevOptions.tsx

```diff
@@ -12,7 +12,7 @@ const DevOptions: React.FC = () => {
 
   const { filteredTemplates } = useListTemplates(templatePermissions, isLoading)
   return (
-    <div style={menuStyle}>
+    <div id="dev-options" style={menuStyle}>
```

Comment:
Still using inlineStyles here?

Reply:
I guess it makes sense to leave this out of the "proper" CSS since it will be removed for production.

Reply:
Ok